src/content/features/add-sidebar-matches-elo.js

```diff
@@ -30,7 +30,7 @@ export default async () => {
   const isFreeMember = getIsFreeMember(self)
 
   let matches = await getPlayerMatches(self.guid, game)
-  matches = mapMatchesWithElo(matches, game)
+  matches = await mapMatchesWithElo(matches, game)
```

Comment:
I have no sidebar matches, so I cannot verify that this works. would like @timche or someone else to please verify that this works, and that is also works with hub matches.

Reply:
Here's how it looks on my end. Note the matches without ELO change are hub matches.
https://i.imgur.com/c7noN92.png